src/BehaviorsSDKManaged/Microsoft.Xaml.Interactivity.Design/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyTitle("Microsoft.Xaml.Interactivity.Design")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyCompany("")]
```

Comment:
Why was this done?

Reply:
Oops, didn't mean to! Fixed, and also centralized the files to remove duplication 🙂